packages/gatsby-transformer-asciidoc/src/gatsby-node.js

```diff
@@ -106,7 +106,7 @@ async function onCreateNode(
   }
 }
 
-const processPluginOptions = _.memoize((pluginOptions, pathPrefix) => {
+const processPluginOptions = _.memoize((pluginOptions, pathPrefix, node) => {
```

Comment:
I don't have enough context on this plugin but this change makes memoization useless, right? Should we pass `baseDir` directly here vs `node`?

Reply:
Yeah, this was a bad change. I'm reverting this and just adding `base_dir` after this function has been called.

`memoize` is the right choice here for plugin speed.